channels/telegram/src/main/kotlin/com/justai/jaicf/channel/telegram/TelegramReactions.kt

```diff
@@ -18,11 +19,34 @@ class TelegramReactions(
 ) : Reactions() {
 
     val chatId = request.chatId
+    private val messages = mutableListOf<Message>()
+
+    private fun addResponse(pair: Pair<retrofit2.Response<Response<Message>?>?, Exception?>) {
+        pair.first?.body()?.result?.let { messages.add(it) }
+    }
 
     override fun say(text: String): SayReaction {
         return sendMessage(text)
     }
 
+    override fun buttons(vararg buttons: String): ButtonsReaction {
+        messages.lastOrNull()?.let { message ->
+            val keyboard = message.replyMarkup?.inlineKeyboard?.toMutableList() ?: mutableListOf()
```

Comment:
As I see, `kotlin-telegram-bot` supports only inline keyboard, but it'd be better to support basic telegram `ReplyKeyboard`
(not much related to this PR, just thoughts for the future)

Reply:
Telegram allows attaching only inline buttons editing the message. The user [must pass keyboard](https://core.telegram.org/bots#keyboards) with the message to display a "basic" keyboard.

Reply:
Yes, you are right, thanks